lib/python/rose/popen.py

```diff
@@ -175,29 +175,31 @@ def run_bg(self, *args, **kwargs):
         stdin = kwargs.get("stdin")
         if isinstance(stdin, str):
             kwargs["stdin"] = PIPE
+        elif stdin is None:
+            kwargs["stdin"] = open(os.devnull)
```

Comment:
There shouldn't be the risk of editors being called via RosePopen, is there any other foreseeable issue?